flake.nix

```diff
@@ -78,7 +76,7 @@
             inherit pkgs;
             ligo =
               if system == "x86_64-linux"
-              then ligo.packages.${system}.ligoLight
+              then pkgs.ligo
```

Comment:
I think this could be changed to just always be `pkgs.ligo` since the one in `nixpkgs` works on both m1 and linux

Reply:
I did it. Please check my other comment